src/lib/Image.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 
-const Img = ({ src, variant, className, onLoad }) => {
+const Img = ({ src, variant, className, onLoad, alt = null }) => {
+  const imgAlt = alt !== null ? alt : src.alt;
+
   if (typeof src === 'object' && !('name' in src)) console.warn(`WARNING: Image with empty {}: ${variant} ${className || ''}`);
 
-  if (typeof src === 'object' && ('name' in src) && (src.name.includes('.svg') || src.name.includes('.gif'))) return (<img src={src.urls.original} alt={src.alt} className={className} onLoad={onLoad} />);
-  if (typeof src === 'object' && ('name' in src)) return (<img src={src.urls[variant]} alt={src.alt} className={className} onLoad={onLoad} />);
-  if (typeof src === 'string') return (<img src={src} alt={src} className={className} onLoad={onLoad} />);
+  if (typeof src === 'object' && ('name' in src) && (src.name.includes('.svg') || src.name.includes('.gif'))) return (<img src={src.urls.original} alt={imgAlt !== null ? imgAlt : src.name} className={className} onLoad={onLoad} />);
+  if (typeof src === 'object' && ('name' in src)) return (<img src={src.urls[variant]} alt={imgAlt !== null ? imgAlt : src.name} className={className} onLoad={onLoad} />);
+  if (typeof src === 'string') return (<img src={src} alt={imgAlt !== null ? imgAlt : src.name} className={className} onLoad={onLoad} />);
```

Comment:
Here it will error out since src is a string. Skip ALTs for string `src`s (the placeholders).